web/packages/extension/src/background.ts

```diff
@@ -9,6 +9,100 @@ async function contentScriptRegistered() {
 }
 
 async function enable() {
+    // Unlike other browsers, Chrome registers rules with an unsupported condition.
+    // Therefore, check that Chrome is a version that supports the responseHeaders condition,
+    // so the browser in use throws an exception or supports the responseHeaders condition.
+    // The alternative check here is to use feature detection to detect support for something
+    // that strictly greater Chrome versions support, but no such features exist yet.
+    const chrome127OrLess =
+        /Chrome\/([0-9]|1[0-9]|1[0-1][0-9]|12[0-7])\./.test(
```

Comment:
Chrome/0. - Chrome/9.
Chrome/10. - Chrome 99.
Chrome 100. - Chrome 119.
Chrome 120. - Chrome 127.

See https://www.useragentstring.com/pages/Chrome/

All Chromium browsers seem to have Chrome in their user-agent string, which is desired here.

The current Chrome Dev, Chrome 128, supports the responseHeaders condition.